test/e2e/gin/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+	r := gin.Default()
+	r.GET("/hello", func(c *gin.Context) {
```

Comment:
it might be nice if the samples apps used unique endpoint names so we can definitively see the output diff in the fixture